domains/v1/domains_test.go

```diff
@@ -4,52 +4,90 @@ import (
 	"fmt"
 	"math/rand"
 
-	"github.com/cloudfoundry-incubator/cf-test-helpers/workflowhelpers"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	"github.com/cloudfoundry/cf-performance-tests/helpers"
+	"github.com/cloudfoundry/cf-test-helpers/v2/workflowhelpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gmeasure"
 )
 
 var _ = Describe("domains", func() {
 	Describe("GET /v3/domains", func() {
-		Measure("as admin", func(b Benchmarker) {
-			workflowhelpers.AsUser(testSetup.AdminUserContext(), testConfig.BasicTimeout, func() {
-				helpers.TimeCFCurl(b, testConfig.BasicTimeout, "/v3/domains")
-			})
-		}, testConfig.Samples)
 
-		Measure("as regular user", func(b Benchmarker) {
-			workflowhelpers.AsUser(testSetup.RegularUserContext(), testConfig.BasicTimeout, func() {
-				helpers.TimeCFCurl(b, testConfig.BasicTimeout, "/v3/domains")
-			})
-		}, testConfig.Samples)
+		It("gets /v3/domains as admin efficiently", func() {
```

Comment:
No need to refer to /v3/domains in each of your `It` blocks since they're inside a `Describe` block that does that.

Reply:
This It block is used to generate the name of the file in the json output, so I think it is necessary to have it here.